robottelo/ui/subscription.py

```diff
@@ -46,6 +46,7 @@ def delete(self):
         """Deletes Manifest/subscriptions via UI."""
         self.click(locators['subs.manage_manifest'])
         self.click(locators['subs.delete_manifest'])
+        self.wait_until_element_go(locators['subs.manifest_exists'], 900)
```

Comment:
900 seconds is too high value for that waiter, I would suggest not more than 300